### PR TITLE
Added reload when needed in ConfigurationChangesTestCase + several tweaks

### PR DESCRIPTION
**Pull request by honza-kasik**
- `assertSuccess()` on write max-history attribute operation
- max-history value as constant
- added reload after write-atribute/add in `before()`

### CODE REVIEW

**Comment by honza-kasik** [comment]
I will squash it, wait

**Comment by honza-kasik** [comment]
Done!

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/server/configurationchanges/ConfigurationChangesTestCase.java [hunk 1]
```diff
@@ -52,15 +52,14 @@
...
+    private static final Operations operations = new Operations(client);
+    private static final Administration administration = new Administration(client);
     private static Address serviceAddress;
     private static ConfigurationChangesResource changesResource;
 
     @BeforeClass
-    public static void beforeClass() {
-        client = ManagementClientProvider.createOnlineManagementClient();
-        operations = new Operations(client);
+    public static void beforeClass() throws InterruptedException, TimeoutException, IOException {
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/server/configurationchanges/ConfigurationChangesTestCase.java]: No need for the throws as those exceptions are not thrown from the used methods.

**Review by rhatlapa — CHANGES_REQUESTED** [review]
No need for the throws ... in the `beforeClass`

**Comment by honza-kasik** [comment]
Removed exceptions from before class, thanks!

**Comment by rhatlapa** [comment]
Thanks, LGTM now, merging.

**Pull request merged**